src/core/include/mp-units/ext/prime.h

```diff
@@ -32,6 +32,7 @@
 import std;
 #else
 #include <array>
+#include <bits/types.h>
```

Comment:
This is normally not the correct header. A tool like clangd will suggest this include due to the internals of the clangd compiler, which matches in most cases the gcc compiler. 
For most compilers a higher level include is normally need, as this one is an implementation detail of the specific compiler. 
Unfortunately I am not sure what the correct C++ header that must be included

Reply:
I quickly looked at the compile error, it appears that `<cstdint>` is the correct high level include since it complains that `'uint64_t' must be declared before it is used` and that should be found in the mentioned header according to https://en.cppreference.com/w/cpp/types/integer

Reply:
Thanks!  `<cstdint>` is already included, so I guess the error is just spurious.